src/test/munit/config.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:http="http://www.mulesoft.org/schema/mule/http"
+      xmlns:oauth="http://www.mulesoft.org/schema/mule/oauth" xmlns:os="http://www.mulesoft.org/schema/mule/os"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="
+http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/os http://www.mulesoft.org/schema/mule/os/current/mule-os.xsd
+http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+http://www.mulesoft.org/schema/mule/oauth http://www.mulesoft.org/schema/mule/oauth/current/mule-oauth.xsd">
+
+    <munit:dynamic-port propertyName="oauth.server.port" min="1024" max="64000"/>
+
+    <!-- Oauth token endpoint stub -->
+    <http:listener-config name="oauthListenerConfig">
+        <http:listener-connection host="localhost" port="${oauth.server.port}"/>
+    </http:listener-config>
+
+    <http:request-config name="requestConfig">
+        <http:request-connection host="localhost" port="${oauth.server.port}">
+        </http:request-connection>
+    </http:request-config>
+
+    <os:object-store name="authCodeObjectStore" doc:name="Object store" persistent="false" />
+    <os:object-store name="fastExpiryObjectStore" doc:name="Object store" persistent="false" entryTtl="5" expirationInterval="5" expirationIntervalUnit="SECONDS" />
+
+    <oauth:token-manager-config name="authCodeTokenManagerConfig" objectStore="authCodeObjectStore"/>
+    <oauth:token-manager-config name="clientCredsTokenManagerConfig"/>
+
+    <http:request-config name="requestConfigWithOAuth">
+        <http:request-connection host="localhost" port="${oauth.server.port}">
+            <http:authentication>
+                <oauth:client-credentials-grant-type
+                        clientId="ndli93xdws2qoe6ms1d389vl6bxquv3e"
+                        clientSecret="yL692Az1cNhfk1VhTzyx4jOjjMKBrO9T"
```

Comment:
About this: I think we should change this back to the hardcoded clientId and clientSecret as these are not sensitive credentials, they are used to connect to a local oauth server. The jenkins job for this module runs on the mule runtime pipeline and it doesn't have vault support like the connectivity pipeline does. Because of this, it is imposible to download the credentials from vault using the connector-maven-plugin.